apiclient/client.go

```diff
@@ -36,16 +36,6 @@ const (
 	DefaultTimeout = 10 * time.Second
 )
 
-// DefaultAPIUrls is a map of default API URLs for each network.
```

Comment:
Isn't this map used for the end2end test or something else?

Reply:
nope. i added this hack as part of my "statesync v2" and i'm happy to revert it TBH